File: app/src/main/java/com/abhi/lokaldemo/api/repo/PicSumService.java
package com.abhi.lokaldemo.api.repo;

import com.abhi.lokaldemo.api.model.PostResponse;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface PicSumService {

    String BASE_URL = "https://picsum.photos/";

    @GET("list")
    Call<List<PostResponse>> getPosts();

}

File: app/src/main/java/com/abhi/lokaldemo/api/repo/PicSumService.kt
package com.abhi.lokaldemo.api.repo

import com.abhi.lokaldemo.api.model.PostResponse
import retrofit2.Call
import retrofit2.http.GET


internal interface PicSumService {

    @GET("list")
    fun getPosts(): Call<List<PostResponse>>

    companion object {
        val BASE_URL = "https://picsum.photos/"
    }
}